docs/text-area.md

```diff
@@ -0,0 +1,113 @@
+# Textarea
+
+Adapt UI provides textarea with three styles and six interaction
+states. Use this component to receive information from the user.
+
+![SCR-20221115-utg](https://user-images.githubusercontent.com/20360870/201976908-c51596e9-0ac6-4740-a2ba-660e6f9d8ca7.png)
```

Comment:
A better preview, taken in a landscape, With the heading and the hamburger menu, would be nice to have.

Reply:
![SCR-20221116-gpf](https://user-images.githubusercontent.com/20360870/202102118-cc64bbf4-020b-4b72-9470-28645159d1a5.png)

Reply:
![SCR-20221116-gzi](https://user-images.githubusercontent.com/20360870/202104722-6d5cf79a-60a7-4a06-ac61-b92aa086bf4c.png)